src/kpatch_elf.c

```diff
@@ -966,3 +967,35 @@ int kpatch_elf_load_kpatch_info(struct object_file *o)
 	kpdebug("failed\n");
 	return -1;
 }
+
+int kpatch_unpatch_find_address(struct object_file *o, unsigned long address)
+{
+	struct kpatch_info *info = o->info;
+	size_t i, ninfo = o->ninfo;
+	for (i = 0; i < ninfo; i++)
+		if (address >= info[i].saddr && address < info[i].saddr + info[i].slen)
+			return 1;
+	return 0;
+}
+
+int print_address_closest_func(struct object_file *o, unw_cursor_t *cur, char *address_str, int strsize)
```

Comment:
This function has nothing to do with ELF. Please place it a different file. kpatch_user.c should be the right place --- it is bloated at the moment but we will deal with that later.